collect.py

```diff
@@ -222,10 +226,120 @@ def get_domtree(driver, file_path):
         f.write(driver.execute_script('return document.documentElement.outerHTML'))
 
 
+def get_coordinates(driver, file_path):
+    dom_tree = etree.HTML(driver.execute_script('return document.documentElement.outerHTML'))
+    dom_element_tree = etree.ElementTree(dom_tree)
+    loc_dict = {}
+    dom_tree_elements = [elem for elem in dom_tree.iter(tag=etree.Element)]
+    web_elements = driver.find_elements_by_css_selector('*')
```

Comment:
Can you use the same find_elements we use in the part of the crawler where we search for all elements on the page to click on them?

Reply:
>  buttons = body.find_elements_by_tag_name('button')
    links = body.find_elements_by_tag_name('a')
    inputs = body.find_elements_by_tag_name('input')
    selects = body.find_elements_by_tag_name('select')
    children = buttons + links + inputs + selects

I think you are talking about this? But we want all elements and not just these.

Reply:
I thought we were also getting all elements somewhere, but my memory failed me :P